### ISSUE TITLE
Have lektor installed via chocolatey

### ISSUE THREAD

**Issue by nimishbongale** [lektor/lektor#814]
Chocolatey is a great package manager for windows. A simple command such as `choco install xxx` will install the necessary software and update system paths automatically.

It would be great to have lektor installed via chocolatey! I do understand the installation isn't exactly an issue right now, but it would be nice to have it this way.

- https://chocolatey.org/
- https://chocolatey.org/docs/create-packages

I'd be interested in implementing this.

**Comment by runfalk** [comment]
I think this is a great idea. If you start implementing this think of the following as requirements:

* The package build scripts must be part of this repository
* There must be a GitHub CI action that builds the package and makes it available as an artifact
* There must be a GitHub CI action that tries to install the package from the previous step (and run something like `lektor --version` to check that it works)
* There must be a GitHub CI action that publishes the updated package on release tags

I think these steps are very important as we would not want to make these packages available and then drop support because they decay over time.

I think we should also make the package depend on ImageMagick and ffmpeg to make sure that the install has access to all functionality by default.

**Comment by nimishbongale** [comment]
@runfalk 

I'll be sure to try it out!

**Comment by nimishbongale** [comment]
Hello @runfalk 

- https://chocolatey.org/packages/lektor/1.0.0

Lektor can now be installed via chocolatey!

`choco install lektor --version=1.0.0`

**What's done**
- lektor's _latest_ version is now available for download from chocolatey
- By default, it adds me as a package maintainer over on chocolatey for lektor (since I created the package). If the maintainers here could create their accounts on chocolatey I'll add you people and remove myself
- Will raise a PR shortly with the code present under a `chocolatey` folder  

**What's pending**
- Will need to setup Github CI scripts to check for a success/failure of chocolatey deploy
- The package currently does not pass package testing over on chocolatey, but I should have that sorted
- Versioning will also need to be taken care of

**Comment by runfalk** [comment]
Hi @nimishbongale,

Nice. I don't have Windows so I can't test anything.

The thing that stand out to me is that it pulls the latest version always, and that the package version is not coupled with the version it actaually installs. If we for instance want to require a different Python version the installer at Chocolatey could fail until we manually update it. It may also lead to confusing bug reports where the users say they are running 1.0.0, or that they are reporting the package because they don't get the latest version.

Is this something that could be changed you think?

**Comment by nimishbongale** [comment]
@runfalk 

Yes. We can configure that accordingly. All that I've done until now is to have lektor installed via chocolatey. By default, chocolatey thinks its `v1.0.0`. This will be configured by me soon!

I just thought once the packages passes all the tests on chocolatey, we can go forward with the versioning bit. We can also automate the versioning and everything using Github CI scripts.

**Comment by nimishbongale** [comment]
@runfalk 

Lektor has passed testing on chocolatey! 🚀 

- https://chocolatey.org/packages/lektor/1.0.0

I'll go ahead and raise a PR and we can talk about the rest there!
